### PR TITLE
2644 small UI tweaks in ACL Editor

### PR DESCRIPTION
**Pull request by UrsaStutsman**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala [hunk 1]
```diff
@@ -117,12 +117,50 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures
...
             aclEditor.canComputeChecked shouldBe false
           }
         }
 
       }
 
+      "should see can compute permission change for users when role changed from writer to reader" in withWebDriver {implicit driver =>
+        val users = UserPool.chooseStudents(2)
+        val user1 = users.head
+        val user2 = users(1)
```
Comment (MattPutnam) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceSpec.scala]: I think this should work:

```scala
val Seq(user1, user2) = UserPool.chooseStudents(2)
```

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/AclEditor.scala [hunk 2]
```diff
@@ -41,6 +41,16 @@ class AclEditor(implicit webDriver: WebDriver) extends OKCancelModal {
...
 
+  def updateAccess(accessLevel: String): Unit = {
+    roleDropdown.select(accessLevel)
+  }
+
+  def removeCanCompute = canComputeBox.ensureUnchecked()
+
+  def canShareChecked: Boolean = canShareBox.isChecked
+
+  def canShareEnabled: Boolean = canShareBox.isEnabled
```
Comment (MattPutnam) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/AclEditor.scala]: If there are so many passthroughs like this, I wonder if the components should just be public and be accessed directly.

File: src/cljs/main/broadfcui/page/workspace/summary/acl_editor.cljs [hunk 3]
```diff
@@ -120,23 +120,25 @@
                  :disabled disabled?
                  :data-test-id (str "role-dropdown-" (not (:read-only? acl-entry)))
                  :value (:accessLevel acl-entry)
-                 :onChange #(swap! state assoc-in [:non-project-owner-acl-vec i :accessLevel] (.. % -target -value))}
+                 :onChange (fn [event]
+                             (let [new-access-level (aget event "target" "value")]
+                               (swap! state update-in [:non-project-owner-acl-vec i] assoc  :accessLevel new-access-level :canCompute (common/access-greater-than-equal-to? new-access-level "WRITER"))))}
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/workspace/summary/acl_editor.cljs]: Line is kinda long. I like starting a new line after the `assoc`.

**Review by MattPutnam — APPROVED** [review]
👍, a few minor comments

**Review by MatthewBemis — APPROVED** [review]

**Pull request merged**